pulsar/consumer_test.go

```diff
@@ -384,6 +385,138 @@ func TestPartitionTopicsConsumerPubSub(t *testing.T) {
 	assert.Equal(t, len(msgs), 10)
 }
 
+type TestActiveConsumerListener struct {
+	lock             sync.RWMutex
+	nameToPartitions map[string]map[int32]struct{}
+}
+
+func (t *TestActiveConsumerListener) getConsumerCount() int {
+	t.lock.RLock()
+	defer t.lock.RUnlock()
+	return len(t.nameToPartitions)
+}
+
+func (t *TestActiveConsumerListener) getPartitionCount(consumerName string) int {
+	t.lock.RLock()
+	defer t.lock.RUnlock()
+	return len(t.nameToPartitions[consumerName])
+}
+
+func (t *TestActiveConsumerListener) BecameActive(consumer Consumer, topicName string, partition int32) {
+	fmt.Printf("%s become active on %s - %d\n", consumer.Name(), topicName, partition)
```

Comment:
Better to use the `Logf` method in `testing.T`

Reply:
Thanks for your suggestion! In this callback, we cannot get `testing.T` so I just use print. But I change other cases to use `Logf`.

Reply:
Oh, I can pass `testing.T` to the callback to use it 😂 Done.